### PR TITLE
Top 3 Scenarios Where Managing Multiple Clusters with One OpenSearch Dashboard is a Life Saver

### LINKED ISSUE #2812: [Feature] Embrace multiple data source in OpenSearch Dashboards
**Describe the blog post your would like to write**
In 2.4 release, multiple data source feature was introduced together with data source client as well as data source management portal to allow users to add new data source to the dashboards and query local and remote OpenSearch clusters. 

In 2.13 release, we introduced a feature in data source plugin to allow external dashboards plugins to register their API schema, and in 2.14 release, to ensure users that have been using the multiple data source feature to have a consistent experience in the dashboards, we enabled multiple data source at 8 dashboards plugins including anormaly detection, maps, and enabled multiple data source at 2 core plugins including TSVB and Region Map. 

This blog would describe how users can interact with multiple data source from core or external plugins.

**What is the title of the blog post?**
Embrace multiple data source in OpenSearch Dashboards

**Who are the authors?**
Lu Yu
**What is the proposed posting date?**
2.14 release

### PR DESCRIPTION
**Pull request by BionIT**
### Description
This blog provides a quick glance at the accumulated features of multi-data sources from version 2.4 to 2.14 and how these features empower your productivity with unified experience. 
 
### Issues Resolved
#2812

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by BionIT** [comment]
@hdhalter @pajuric This PR has been open for 5days, and it will be part of 2.14 release. Could you help with next steps?

**Comment by hdhalter** [comment]
> @hdhalter @pajuric This PR has been open for 5days, and it will be part of 2.14 release. Could you help with next steps?

Sure, @BionIT, I'll have a tech writer review it. In the meantime, can you please fix the Vale errors?

**Comment by natebower** [comment]
https://github.com/opensearch-project/documentation-website/issues/7139

**Comment by natebower** [comment]
Per @pajuric, now being reviewed by @brijos: https://github.com/opensearch-project/documentation-website/issues/7139#issuecomment-2110646779

**Comment by natebower** [comment]
Per @brijos, he will work with the appropriate technical writer to revise the draft and then send for editorial review.

**Comment by BionIT** [comment]
Thanks @hdhalter and @natebower ! I will address the Vale errors in the meanwhile

File: _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md
Comment (natebower) [inline _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md:64]:
```suggestion
You can also manage notification channels, as shown in the following image.
```

File: _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md
Comment (natebower) [inline _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md:68]:
```suggestion
Users can manage remote ML models, as shown in the following image. The pretrained models are deployed to the selected cluster through the Dev Tools console. 
```

File: _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md
Comment (natebower) [inline _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md:72]:
```suggestion
The Dev Tools console, shown in the following image, is integrated with the multiple data sources feature. You can select the target data source and execute commands to query to modify the backend data directly from a central location. 
```

Comment (natebower) [inline _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md:72]: In the second sentence, we need to choose either "to query" or "to modify".

File: _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md
Comment (natebower) [inline _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md:76]:
```suggestion
The multiple data sources feature enables OpenSearch Dashboards to provide a centralized interface for creating, viewing, and modifying anomaly detectors from various OpenSearch clusters, streamlining the management process, as shown in the following image.  
```

File: _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md
Comment (natebower) [inline _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md:83]:
```suggestion
For users with multiple OpenSearch clusters handling search traffic, the multiple data sources feature allows OpenSearch Dashboards to compare search results across these clusters within a single instance by using the data source picker in the Compare search results application.
```

Comment (natebower) [inline _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md:83]: Should we format "Compare search results" in such a way as to indicate that it's the proper name of something?

File: _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md
Comment (natebower) [inline _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md:93]:
```suggestion
Special thanks to Ramakrishna Chilaka, Prabhat Sharma, Derek Ho, Junqiu Lei, Jackie Han, Riya Saxena, Sumukh Hanumantha Swamy, Lin Wang, Ella Zhu, Emma Jin, Huy Nguyen, and Zhongnan Su for their work in building multiple data sources support for OpenSearch Dashboards.
```

**Review by natebower — COMMENTED** [review]
@BionIT @vagimeli @brijos Editorial review complete. Please see my comments and changes and let me know if you have any questions. Provided there are no substantial changes or additions, I should not need to re-review, but I'm happy to review/verify anything about which you're unsure. Thanks!

File: _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md
Comment (BionIT) [reply _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md]: It is not a name, could you clarify?

File: _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md
Comment (BionIT) [reply _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md]: Do you have a suggestion? Visually, it will show the data sources being used by current map across multiple layers

File: _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md
Comment (natebower) [reply _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md]: In that case, "compare" should be lowercase.

File: _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md
Comment (BionIT) [reply _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md]: Right, will remove `to modify`

File: _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md
Comment (natebower) [reply _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md]: In that case, I think "indicated" is probably fine 😄

File: _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md
Comment (BionIT) [reply _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md]: Good catch, it should be region map, changed in new commit

File: _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md
Comment (natebower) [inline _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md:15]:
```suggestion
The multiple data sources feature is available in OpenSearch versions 2.4 and later. The feature supports authentication types (AWS Signature Version 4, basic authentication), Vega visualizations, and data management features (importing of saved objects). With OpenSearch 2.14, we have expanded multiple data sources support for nine dashboard features (index management, machine learning [ML], search relevance, anomaly detection, maps, security, notifications, Query Workbench, and trace analytics) and certain visualizations (time-series visual builder [TSVB] and region map). If you are managing an environment with multiple clusters and use one cluster more often than the others, you now have the ability to set a default cluster that can be referenced by all Dashboards features.
```

File: _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md
Comment (BionIT) [reply _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md]: sorry for the confusion, it is the title of the page though, I can add tick like `Compare search results`

Comment (BionIT) [reply _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md]: changed to `Compare search results` by adding `

**Review by natebower — DISMISSED** [review]
@BionIT LGTM. Thanks for all of your work on this!

**Comment by natebower** [comment]
@pajuric Over to you. Thanks!

File: _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md
Comment (pajuric) [inline _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md:3]: lifesaver is one word.

Comment (BionIT) [reply _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md:3]: > lifesaver is one word.

Is this the only change needed before merging in this PR?

Comment (BionIT) [reply _posts/2024-05-16-Top-3-Scenarios-Where-Managing-Multiple-Clusters-with-One-OpenSearch-Dashboards-is-a-Life-Saver.md:3]: Synced offline with @pajuric , maintainers would help to change the title to make lifesaver as one word. PR can be published tomorrow

**Review by nateynateynate — APPROVED** [review]
Date edited, lifesaver becomes one word. We're set.

**Pull request merged**
